build.gradle

```diff
@@ -27,6 +27,11 @@ repositories {
 }
 
 configurations {
+    all {
+        // Exclude JUnit 4. Only JUnit 5 is used
+        exclude group: 'junit', module: 'junit'
+        exclude group: 'org.junit.vintage', module: 'junit-vintage-engine'
+    }
```

Comment:
Note, junit:junit dependency comes from biz.paluch.logging:logstash-gelf dependency
`./gradlew dependencies  | grep -A 10 -B 10 junit`
`biz.paluch.logging:logstash-gelf:1.5.1` is  really old: https://github.com/mp911de/logstash-gelf/releases/tag/logstash-gelf-1.5.1